http-transport-binding.md

```diff
@@ -0,0 +1,363 @@
+# HTTP Transport Binding for CloudEvents
+
+## Abstract
+
+The HTTP Transport Binding for CloudEvents defines how events are mapped to
+HTTP 1.1 request and response messages.
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+- 1.1. [Conformance](#11-conformance)
+- 1.2. [Relation to HTTP](#12-relation-to-http)
+- 1.3. [Content Modes](#13-content-modes)
+- 1.4. [Event Formats](#14-event-formats)
+- 1.5. [Security](#15-security)
+2. [Use of CloudEvents Attributes](#2-use-of-cloudevents-attributes)
+- 2.1. [contentType Attribute](#21-contenttype-attribute)
+- 2.2. [data Attribute](#22-data-attribute)
+3. [HTTP Message Mapping](#3-http-message-mapping)
+- 3.2. [Binary Content Mode](#31-binary-content-mode)
+- 3.1. [Structured Content Mode](#32-structured-content-mode)
+4. [References](#4-references)
+
+## 1. Introduction
+
+[CloudEvents][CE] is a standardized and transport-neutral definition of the
+structure and metadata description of events. This specification defines how
+the elements defined in the CloudEvents specification are to be used in
+[HTTP 1.1][RFC7230] requests and response messages.
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][RFC2119].
+
+### 1.2. Relation to HTTP
+
+This specification does not prescribe rules constraining the use or handling of
+specific [HTTP methods][RFC7231-Section-4], and it also does not constrain the
+[HTTP target resource][RFC7230-Section-5-1] that is used for transferring or
+soliciting events.
+
+Events can be transferred with all standard or application-defined HTTP request
+methods that support payload body transfers. Events can be also be transferred
+in HTTP responses and with all HTTP status codes that permit payload body
+transfers.
+
+All examples herein that show HTTP methods, HTTP target URIs, and HTTP status
+codes are non-normative illustrations.
+
+This specification also applies equivalently to HTTP/2 ([RFC7540][RFC7540]),
+which is compatible with HTTP 1.1 semantics.
+
+### 1.3. Content Modes
+
+This specification defines two content modes for transferring events:
+*structured* and *binary*. Every compliant implementation SHOULD support both
```

Comment:
I wrote "implementation" specifically so that producers and consumers have a choice when they use said implementation. Senders will typically pick one of the two and Consumers should be prepared to understand both. Do you think that needs rephrasing?

Reply:
@inlined should we open an issue to revisit this? If you have some text in mind perhaps you could PR it?  If you have some questions about the wording then others might as well